tests/polyfill/test_polyfill.py

```diff
@@ -1,42 +1,35 @@
-import h3
 import pytest
 
+import h3
 from h3 import H3ResDomainError
 
 from .. import util as u
 
 
 def get_us_box_coords():
-
     # big center chunk of the US in lat/lng order
     outer = [
         [42.68, -110.61],
         [32.17, -109.02],
-        [31.57,  -94.26],
-        [42.94,  -89.38],
-        [42.68, -110.61]
+        [31.57, -94.26],
+        [42.94, -89.38],
+        [42.68, -110.61],
     ]
 
     hole1 = [
         [39.77, -105.07],
         [34.81, -104.72],
-        [34.77,  -98.39],
-        [40.14,  -96.72],
-        [39.77, -105.07]
+        [34.77, -98.39],
+        [40.14, -96.72],
+        [39.77, -105.07],
     ]
 
-    hole2 = [
-        [41.37, -98.61],
-        [40.04, -91.80],
-        [42.32, -91.80],
-        [41.37, -98.61]
-    ]
+    hole2 = [[41.37, -98.61], [40.04, -91.80], [42.32, -91.80], [41.37, -98.61]]
```

Comment:
Worse

Reply:
Yes. Adding the trailing comma will auto fmt this to be element-per-line.